docs/howto/manage-charms.rst

```diff
@@ -234,12 +234,17 @@ charm's project file, specify an item under the :ref:`links.website
 Add docs
 ~~~~~~~~
 
-If you publish your charm on Charmhub, reference documentation about the charm's
-resources, actions, configurations, relations, and libraries is extracted automatically.
-However, you should also aim to add further docs, e.g., a tutorial, how-to guides, etc.
-To provide a link to these docs, in your charm's project file, specify a
-value for the :ref:`links.documentation <charmcraft-yaml-key-documentation>` key. Note
-that at present this must be a Discourse page. E.g.,
+When you publish your charm on Charmhub, reference documentation about the charm's
```

Comment:
> When

I have a feeling that the "if" was there to suggest that publication to Charmhub wasn't mandatory. We usually keep private stores in mind.

Reply:
Or indeed charms that never go into a store and just get deployed through CI/CD

Reply:
Reverted the changes back to "if".